README.md

```diff
@@ -1,3 +1,33 @@
 # fasjson-client
 
 A python client library for the FASJSON API
+
+This client uses the bravado library to build dynamic api methods based on open-api specs (version 2.0): https://github.com/Yelp/bravado
+
+## Usage
+
+```
+c = Client.from_url('http://myspecurl.com', base_url='http://fasjson.example.test/fasjson' principal='admin@EXAMPLE.TEST')
+
+output = c.api.v1.whoami().response().result
+
+print(output)
+```
+
+## Development
+
+Install dependencies:
+
+```
+pip install -U -r requirements.txt
```

Comment:
A real python package would be better. Could you wite a `setup.py` file or a `pyproject.toml` file?

Reply:
yep, I just sent it like this for an early feedback, "development" here stand for people who want to contribute to the repository.

Reply:
can you send me offline (irc, email etc) what values should I use for author name, email, etc.

Reply:
You can have a look at noggin and / or fasjson if you need inspiration, that's what I would do ;-)

Reply:
Any news on that item?